code/modules/clothing/suits/marine_armor/ert.dm

```diff
@@ -79,6 +79,7 @@
 	item_state = "armor"
 	item_state_slots = null
 	contained_sprite = TRUE
+	slowdown = SLOWDOWN_ARMOR_LIGHT
```

Comment:
This armor already has slowdown by inheritance from pmc/light armor . There is no need to write it again